panacea/web/static/js/view.js

```diff
@@ -58,32 +107,42 @@ const displayWarnings = (container, preambleText, warnings, pml) => {
 
     container.appendChild(wrapper);
   }
+
+  showResults();
 };
 
 export const displayUnnamed = (unnamed, pmlLines) => {
-  const container = document.getElementById('unnamed-text');
-  const preamble = 'I found the following unnamed PML constructs:';
-  const warnings = unnamed.map(({type, line}) => {
-    return [{line, identifier: type}];
-  });
-
-  displayWarnings(container, preamble, warnings, pmlLines);
-  showElement(unnamedPanel);
+  if (unnamed.length > 0){
+    const container = document.getElementById('unnamed-text');
+    const preamble = 'I found the following unnamed PML constructs:';
+    const warnings = unnamed.map(({type, line}) => {
+      return [{line, identifier: type}];
+    });
+
+    displayWarnings(container, preamble, warnings, pmlLines);
+    showElement(unnamedPanel);
+  } else {
+    // no unnamed properties to display
```

Comment:
We don't, I put it in there with the idea that we might display some info along the lines that `the thing you're trying to display is empty`. In the spirit of YAGNI I will remove it 👍